firmwares/enbrighten-ge/ZW4002_55258.json

```
@@ -10,10 +10,33 @@
 	],
 	"upgrades": [
 		{
+			"$if": "firmwareVersion >= 5.0",
 			"version": "5.50",
 			"changelog": "1. Original Release Firmware",
 			"url": "https://raw.githubusercontent.com/jascoproducts/firmware/9b5520f5056f7d3fce3fa6adc28bdaf27f8d0a68/zwave/Enbrighten-GE/ZW4002/55258%20-%20In-Wall%20Smart%20Fan%20Speed%20Control%2C%20500S/5.50/ZW4002_Enbrighten-GE_55258_5.50.otz",
 			"integrity": "sha256:59ae4c4f749d3ad22b1d490e9dd0afe855bbd9914337f37a41beb56d62c638b2"
+		},
+		{
+			"$if": "firmwareVersion === 0.51",
+			"version": "5.50",
```

I'd add
```suggestion
			"version": "5.50",
			"channel": "beta",
```
so this doesn't permanently get shown as an update to the users of the alternate firmware.